src/enums/componentsCompanyInfo.ts

```diff
@@ -19,7 +19,10 @@ export enum ComponentsCompanyInfo {
   'company-provisions',
   'resolution-dates',
   'association-type',
-  'create-special-resolution',
+  'rules',
+  'memorandum',
+  'special-resolution',
+  'special-resolution-signature',
```

Comment:
These must be in same order as in src/store/state/state-model.ts.

Reply:
Oh man, will fix

Reply:
Fixed, thanks.